experimental/unitsconversion/src/provider.rs

```diff
@@ -41,26 +26,8 @@ pub struct UnitsConstantsV1<'data> {
     #[cfg_attr(feature = "serde", serde(borrow))]
     pub constants_map: ZeroMap<'data, str, u16>,
```

Comment:
Suggestion: use a `ZeroMap<'data, str, str>`; you only need to split the keys and values into separate tables if you're doing something fancier like deduplication.

Reply:
Okay, I added to the todo issue: https://github.com/unicode-org/icu4x/issues/3882